logever/src/main/resources/data.sql

```diff
@@ -40,8 +40,8 @@ WHERE NOT EXISTS (
 ) LIMIT 1 ^;
 
 
-INSERT INTO user (username,password,first_name,department,is_active,created_by,create_dt)
-SELECT * FROM (SELECT 'admin' as username,'$2a$10$neAtxbkTe3P1lUkvghFG0e9tO7Lfx47i4wuvl/UJqPSqF5lvpgRBa','ADMIN' as fname,'ZYDUS',true,'admin',current_timestamp) AS tmp
+INSERT INTO user (username,password,first_name,department,date_of_birth,is_active,created_by,create_dt)
```

Comment:
How will existing data be impacted with this new field? Won't we need to alter the table as well?